### ISSUE TITLE
Implement SET_TEXT_DROPSHADOW in ScaledText

### ISSUE THREAD

**Issue by justalemon** [LemonUIbyLemon/LemonUI#169]
SET_TEXT_DROPSHADOW allows you to make a more customizable drop shadow than SET_TEXT_DROP_SHADOW.

**Comment by kagikn** [comment]
I noticed this issue now, but you assumed wrong. `SET_TEXT_DROPSHADOW` just does the same thing as `SET_TEXT_DROP_SHADOW` unless a non-positive value is passed as the 1st argument. The 2nd to 5th argument is marked as `UNUSED_PARAM` in `commands_hud.cpp` and the native implementation doesn't even use none of them. If the 1st arg is not positive, the drop shadows is removed for the next text draw.

I wish I noticed the wrong assumption before releasing any stable releases tbh...
